app/src/pages/index.tsx

```diff
@@ -16,11 +16,8 @@ const Home: NextPage = () => {
 };
 
 export const getServerSideProps: GetServerSideProps = async ({ locale }) => {
-  return {
-    props: {
-      ...(await serverSideTranslations(locale || "en", ["common"])),
```

Comment:
ℹ️ The namespaces argument is optional, and by default it will load all namespaces. I think restricting which namespaces are loaded can be a project-level optimization if the project introduces multiple namespaces.